connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -1324,7 +1353,6 @@ private static <T> void waitForConsumingAllRecords(Consumer<T, T> consumer, int
     private static Map<String, String> basicMM2Config() {
         Map<String, String> mm2Props = new HashMap<>();
         mm2Props.put("clusters", PRIMARY_CLUSTER_ALIAS + ", " + BACKUP_CLUSTER_ALIAS);
-        mm2Props.put("max.tasks", "10");
```

Comment:
Unrelated change: this property has no effect and should be removed so as not to cause FUD.